src/v0/safes.ts

```diff
@@ -46,6 +46,19 @@ export interface Safe {
   active?: boolean
 }
 
+export interface BookRequestBody {
+  to: string,
+  from: string,
+  issuer: string
+  items: Array<Object>,
+  comment?: string,
+  initiated_at?: string
+}
+
+export interface BookResponse {
```

Comment:
You don't use this type interface anywhere.

Reply:
You are right, it is leftover code. I overlooked this.

Reply:
So just remember to remove it in your next changes to this file.

Reply:
Fixed in PR #176